src/libsystemd-network/sd-dhcp-server-lease.c

```diff
@@ -204,3 +204,69 @@ int sd_dhcp_server_set_static_lease(
         TAKE_PTR(lease);
         return 0;
 }
+
+static int dhcp_server_lease_append_json(sd_dhcp_server_lease *lease, JsonVariant **array) {
+        assert(lease);
+        assert(array);
+
+        return json_variant_append_arrayb(
+                        array,
+                        JSON_BUILD_OBJECT(
+                                        JSON_BUILD_PAIR_BYTE_ARRAY(
```

Comment:
btw, why not use JSON_BUILD_BASE64() for this? I think this is actually more in line how most web people would encode binary blobs.

Reply:
(i.e. for new formats i'd always prefer base64 strings in stead of byte arrays to encode binary blobs on json)

Reply:
ahh, this is not a new format i guess... then ignore my comment